DEVELOPMENT.md

```diff
@@ -93,25 +119,66 @@ After which, you can rerun a `px auth login` to authenticate with the dev cloud
 Make sure to `px delete --clobber` if running a prior instance of Vizier pointing to another cloud instance.
 
 ## Vizier
+Vizier is Pixie’s data collector that runs on each cluster. It is responsible
+for query execution and managing PEMs.
+
+### Getting Started
+Verify that you can build Pixie Vizier and run the unit tests.
+
+The following will build Pixie Vizier and then run the unit tests for the UDF
+(User Defined Function) math operations. These UDFs will be available directly off
+the px object inside a PxL script. If you are running the development environment via
+Docker, the following should be run inside the Docker container.
```

Comment:
I think this may be more information than is needed for people not working on UDFs, what about, 

"The following will run an example unit test in Pixie's Vizier module."